### PR TITLE
Added cargo guidebook entry and updated guidebook tree to match

### PR DESCRIPTION
**Pull request by Timemaster99**
## About the PR 

Added a guidebook entry for cargo with a complimentary shuttle piloting guide. Assumes the reader has knowledge of controls and nothing else. Is comprehensive, and can be edited in the future to add tips if need be. Equipment section is copied from the wiki.

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added cargo guidebook entry.

image
image

### CODE REVIEW

File: Resources/Server Info/Guidebook/Cargo.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<Document>
+  # Cargo
+  The Quartermaster is responsible for managing the station's cargo bay, ensuring that cargo is bought and unnecessary items are sold. They are also responsible for managing the cargo bank account and shuttle.
+
+  ##Your Equipment
+  <Box>
+  <GuideEntityEmbed Entity="AppraisalTool"/>
+  </Box>
+  As a Quartermaster your equipment is very basic, though you do come with an [color=#a4885c]appraisal tool[/color] in your pocket. This appraisal tool allows you to scan items about the station and determine their value. This will allow you to determine some starting items to sell to raise your money.
```
Comment (Torun-S) [inline Resources/Server Info/Guidebook/Cargo.xml]: Punctuation, first sentence.

File: Resources/Server Info/Guidebook/Cargo.xml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  <GuideEntityEmbed Entity="CrateGeneric"/>
+  </Box>
+  <Box>
+  <GuideEntityEmbed Entity="CrateEngineering"/>
+  </Box>
+  <Box>
+  <GuideEntityEmbed Entity="CrateSecgear"/>
+  </Box>
+  </Box>
+  When the shuttle is docked, it's time to unload. Drag the boxes out and make sure to deliver them to their respective department. Each box will have a label attached to it that can be removed by right clicking and selecting remove label (or leaving it on there if you are lazy, as most are).
```
Comment (Torun-S) [inline Resources/Server Info/Guidebook/Cargo.xml]: Color "remove label" in beige.

File: Resources/Server Info/Guidebook/Cargo.xml
Comment (Torun-S) [inline Resources/Server Info/Guidebook/Cargo.xml:1]: What you can do is take all the text in parentheses and color it in darker gray. You can find the exact hex value elsewhere in the guidebook, like in cookbook.xml.

**Review by Torun-S — COMMENTED** [review]
Will work, can polish.

**Comment by Timemaster99** [comment]
Should be all fixed up now.

**Review by Torun-S — APPROVED** [review]
Cool! Thanks for putting in the work!

**Pull request merged**